manifests/00-crd-extension-console-link.yaml

```
@@ -54,3 +54,17 @@ spec:
               type: string
               description: Determines which dropdown menu the link is added to (HelpMenu, UserMenu)
               pattern: ^(HelpMenu|UserMenu)$
+            applicationMenu:
+              type: object
+              description: Object that holds application menu defaults if the location is set to ApplicationMenu
```

Are these really "defaults"? Maybe

```suggestion
              description: Object that holds application menu properties if the location is set to ApplicationMenu
```

---

Agree, much better.